src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -226,7 +226,12 @@
                                 (or (not public)
                                     (common/access-greater-than-equal-to? accessLevel "WRITER")))}
                   {:label "Public Workspaces"
-                   :predicate :public}]}
+                   :predicate :public}
+                  {:label "Featured Workspaces"
+                   :predicate (fn [{:keys [workspace]}]
+                                (= (count (filterv (fn [featured-workspace]
+                                  (and (= (:name workspace) (:name featured-workspace))
+                                        (= (:namespace workspace) (:namespace featured-workspace)))) (:featured-workspaces props))) 1))}]}
```

Comment:
Convert `featured-workspaces` to a set upon loading, then this predicate can just be:
```clojure
(fn [workspace]
  (contains? (:featured-workspaces props) (ws-common/workspace->id workspace)))
```

Reply:
Thanks, this is really nice.

I have to rein in the part of my brain that wants to write this:
```
@Override
public boolean equals(WorkspaceRef other) {
    return this.name.equals(other.name) && this.namespace.equals(other.namespace);
}
```